src/frontend/src/pages/AdminToolsPage/TeamsTools.tsx

```diff
@@ -131,18 +144,60 @@ const TeamsTools = () => {
               />
             </FormControl>
             <FormControl fullWidth>
-              <FormLabel>Description</FormLabel>
-              <ReactHookTextField
+              <Controller
                 name="description"
                 control={control}
-                fullWidth
-                multiline
-                rows={5}
-                errorMessage={errors.description}
-                maxLength={300}
+                rules={{ required: true }}
+                render={({ field }) => (
+                  <Box sx={{ display: 'flex-col' }}>
```

Comment:
'display: flex-col' isn't actually doing anything, I believe this is from tailwind? We don't use that, so it's not actually applying any css